main/guides/zoe/contract-basics.md

```diff
@@ -1,46 +1,56 @@
 # Smart Contract Basics
 
 Before we look at how to make a contract such as the one in [the
-basic dapp](../getting-started/) in the previous section, let's cover some basics.
+basic dapp](../getting-started/) in the previous section, let's cover some basics by writing a simple contract that returns a greetings message. We will simply call it _greetings smart contract_. 
 
-A contract is defined by a JavaScript module that exports a `start` function
-that implements the contract's API.
+A contract is defined by a JavaScript module that exports a `start` function. For our greetings smart contract, the declaration of `start` function looks like this:
 
 <<< @/../snippets/zoe/src/01-hello.js#start
 
-Let's start with a contract with a simple `greet` function:
+Apart from `start`, we can have extra functions in our contract. These functions may be defined outside or within the `start` function depending on whether they need access to the variables defined in the `start` function.  For the greetings smart contract, we will have a simple `greet` function (outside `start`) that takes a string as a parameter (for example, name of the person calling the function) and returns a customized greeting message.
 
 <<< @/../snippets/zoe/src/01-hello.js#greet
 
-The `start` function can expose the `greet` function
-as part of the contract API by making it
-a method of the contract's `publicFacet`:
+The `greet` function, along with any other function besides `start`, must be exposed through the `start` function before it can be called. This is accomplished using a special object called `publicFacet`, which is returned by the `start` function. In the greeting contract, the `start` function exposes the `greet` function by defining it as a method of the contract's `publicFacet`, as shown below:
```

Comment:
> must be exposed through the `start` function before it can be called.

The paragraph implies that the start function directly exposes other functions through publicFacet. However, the start function is primarily used to initialize the contract and return facets (such as publicFacet and creatorFacet), which can then expose methods like greet.

Reply:
Would it be fine if we removed "through the `start` function " from the first sentence? Note that the audience probably does not know anything about `facet` at this point.

Reply:
After our discussion, I am changing it to the following:

The `greet` function, along with any other public function, must be made accessible through the `publicFacet` of the contract. The `publicFacet` is returned by the `start` function, which initializes the contract. In the greeting contract, the `start` function exposes the `greet` function by defining it as a method of the contract's `publicFacet`, as shown below:

Reply:
TECHNICAL:

"The publicFacet is returned by the start function" is incorrect. The start fuction returns an object with a `publicFacet` property.

Reply:
Fixed!